ui/gtk3/panel.vala

```diff
@@ -1419,6 +1419,12 @@ class Panel : IBus.PanelService {
 
         var engine = m_bus.get_global_engine();
```

Comment:
Thank you for the patch.
I think unowned should be used here.
unowned IBus.EngineDesc engine = m_bus.get_global_engine();